gltf/gltfpack.cpp

```diff
@@ -843,6 +895,16 @@ int gltfpack(const char* input, const char* output, const char* report, Settings
 		settings.texture_embed = true;
 	}
 
+	if (oext && (strcmp(oext, ".vrm") == 0 || strcmp(oext, ".VRM") == 0))
+	{
+		settings.vrm = true;
+		settings.keep_extras = true;
```

Comment:
I would prefer to omit these, esp. `quantize` but preferably others. It's not clear to me why these need to be present, and if a user wants to apply these optimizations to a VRM file there's not going to be a way to do this without adding more kinds of command-line options, if `vrm` extension acts as an override.